content/aws_marketplace.md

```diff
@@ -16,557 +16,120 @@ product = ["client", "workstation"]
+     - Stack Name: `Chef-Automate`
+     - EC2RootVolumeSize: `Default: 40`
+     - Instance Type:`Default: t2.xlarge`
+     - KeyName: _Enter your existing keypair_
+     - SecurityGroupCidrIp: `0.0.0.0/0`
+     - SubnetCIDR: `10.0.0.0/24`
+     - VpcCIDR: `10.0.0.0/16`
 
-### Change the hostname
+1. Select **Next** and create your Chef Automate deployment. This process can take several minutes.
 
-To update the hostname, do the following:
+1. Give Chef Automate an additional five minutes for all the services to start running.
 
-1.  Run `sudo -i` to gain administrator privileges.
+![Select next to create stack](/images/StackDetails.png "Stack Details")
 
-2.  Run `chef-marketplace-ctl hostname` to view the current hostname.
+## Post-Installation
 
-3.  Configure the `api_fqdn` in `/etc/chef-marketplace/marketplace.rb`
+1. Navigate to the AWS deployment **output** tab and locate the Chef Automate URL, user name, and password. You will need these in the next steps.
 
-    ```none
-    echo 'api_fqdn "<new.fully.qualified.hostname.com>"' | sudo tee -a /etc/chef-marketplace/marketplace.rb
-    ```
+![AWS Chef Automate deployment output tab contains your instance URL, user name, and password ](/images/OutputPage.png "Output Page")
 
-4.  Run `chef-marketplace-ctl reconfigure` to update Chef Automate and
-    Chef Infra Server configuration.
+- Open your browser and paste the Chef Automate URL, which will open an alert page.
 
-5.  Run `chef-server-ctl stop` to stop Chef Infra Server.
+- Select **Advanced** and continue.
 
-6.  Run `automate-ctl stop` to stop Chef Automate.
+![Select 'advanced' to bypass the warning that the page is not secure](/images/NotSecurePage.png "Not Secure Page").
 
-7.  Run
-    `chef-marketplace-ctl hostname <new.fully.qualified.hostname.com>`
-    to update the hostname.
+- Enter your **Username** and **Password** and select **Sign In**.
 
-8.  Run `automate-ctl reconfigure` to ensure Chef Automate has been
-    correctly configured with the new hostname.
+![ ](/images/AutomateUI.png "Automate")
 
-9.  Run `chef-server-ctl reconfigure` to ensure Chef Infra Server has
-    been correctly configured with the new hostname.
+- Fill out the registration form and [Accept the Chef License](https://docs.chef.io/chef_license_accept/).
 
-10. Run `automate-ctl restart` to restart Chef Automate
+- Select **Register** to enter Chef Automate.
 
-11. Run `chef-server-ctl restart` to restart Chef server
+![ ](/images/WelcomePage.png "Welcome Page")
 
-### Change instance size
+- Congratulations! You've started Chef Automate!
 
-To edit the Amazon Machine Images (AMI) instance size, do the following:
+![ ](/images/DashboardsPage.png "Dashboards Page")
 
-1.  Login using SSH to access the Chef Automate instance. Use the SSH
-    key pair and the IP address or public hostname that was
-    automatically assigned when the Amazon Machine Images (AMI) was
-    launched. The default user is `ec2-user`. For example:
+## Add Chef Servers
```

Comment:
The ordered list item numbers aren't being formatted correctly by Hugo. The empty line between the starting list item paragraph and the additional text/images of that list item seems to be throwing Hugo off. 

So this isn't working:

```
1. Add Chef-Server Details, select the Add Chef Infra Server Button.

  ![ ](/images/AddChefServer.png "Add Chef Server")
  
1. Enter the server name, FQDN, and IP address. Then select **Add Chef Infra Server** to create the server.  
```


But this does:

```
1. Add Chef-Server Details, select the Add Chef Infra Server Button.
  ![ ](/images/AddChefServer.png "Add Chef Server")
  
1. Enter the server name, FQDN, and IP address. Then select **Add Chef Infra Server** to create the server.  
```